inlong-sdk/sort-sdk/pom.xml

```diff
@@ -111,6 +111,18 @@
             <groupId>org.apache.inlong</groupId>
             <artifactId>sdk-common</artifactId>
             <version>${project.version}</version>
+            <exclusions>
```

Comment:
it's better to remove the PB dependency in `sdk-common`?

Reply:
yes